src/std/rec/Makefile

```diff
@@ -53,3 +53,9 @@ stdRecords_DBD = $(patsubst %,%.dbd,$(stdRecords))
 dbRecStd_SRCS += $(patsubst %,%.c,$(stdRecords))
 
 HTMLS += $(patsubst %.dbd.pod,%.html,$(notdir $(wildcard ../rec/*Record.dbd.pod)))
+
+vpath %.jpg $(USR_VPATH) .. $(SRC_DIRS) ../rec/image
+vpath %.gif $(USR_VPATH) .. $(SRC_DIRS) ../rec/image
```

Comment:
Could you use a modern non-lossy image format instead of `.jpg` – either `.png` or `.svg`, and maybe also convert the `.gif` image into that same format?

Originally you were loading the image files from the image subdirectory; if you write
```Makefile
vpath %.png $(SRC_DIRS)
HTMLS += image/compress-1.png
HTMLS += image/compress-2.png
```
the files should be found and will be installed into the `html/image` subdirectory as well.

Reply:
Thanks! I will change it in next commit.